gitlab/resource_gitlab_project_test.go

```diff
@@ -206,6 +206,129 @@ func TestAccGitlabProject_initializeWithReadme(t *testing.T) {
 	})
 }
 
+func TestAccGitlabProject_templateName(t *testing.T) {
```

Comment:
Can you add these tests as additional TestSteps in the `basic` test above, instead of making them separate tests? That's how Terraform recommends doing it. That way we can reduce some of the testing overhead, and also test what happens when the user updates these attributes. Once you implement the `ForceNew: true` change I suggested in another comment, Terraform will handle destroying and creating the test project for you.